doc/version_history.rst

```diff
@@ -0,0 +1,12 @@
+.. py:currentmodule:: lsst.ts.audio.broadcaster
+
+.. _lsst.ts.audio.broadcaster.version_history:
+
+===============
+Version History
+===============
+
+v0.0.1
```

Comment:
I recommend this be 0.1.0 since we use semantic versioning and for a pre-stable API that's what they recommend how the version should be structured.